applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -639,28 +384,31 @@ static void fix_rep_wk(struct k_work *work)
 		}
 	}
 
+#if defined(CONFIG_SLM_NRF_CLOUD)
 	if (IS_ENABLED(CONFIG_SLM_LOG_LEVEL_DBG)) {
 		goto update_pgps;
 	}
 
+	struct nrf_modem_gnss_nmea_data_frame nmea;
```

Comment:
Please let it be

Reply:
Investigated this a bit, I was in the wrong about some of the coverity issues, declaring variables beyond function start does not cause a coverity issue by itself, it is only caused if there is a goto which skips the initialization of the variable.

This can stay as is and sorry about making you move some of the variables. Some were necessary, but not all.